### ISSUE TITLE
Notification During Background Install

### ISSUE THREAD

**Issue by warmoans** [MuntashirAkon/AppManager#1151]
### Please check before submitting an issue

- [X] I am using the latest version of App Manager
- [X] I have searched the issues and haven't found anything relevant
- [X] I have read the docs

### Describe a description of the new feature

A notification while a background install is happening would be nice. Currently the behavior is a notification after the install finishes.

### Describe the solution you'd like

A notification while a background install is happening.

**Comment by MuntashirAkon** [comment]
Not sure what you're talking about. App Manager does display a notification in the background during an installation.

**Comment by warmoans** [comment]
You're right. I guess the apps I tried it with installed so quickly I didn't realize. Just tried with a larger app. Sorry about that.
